oop_hw3_v2/wagon.py

```diff
@@ -0,0 +1,17 @@
+from aqa_course.oop_hw3_v2.train import Train
+
+
+class Wagon(Train):
+    def __init__(self):
+        super().__init__()
+
+    def add_passenger(self, passenger: str):
+        return self.list_of_passengers.append(passenger)
+
+    def __len__(self, arg):
```

Comment:
you make it too complicated.
So you need just get len of the passengers field which is the list in this `__len__` magic method